server/controllers/UserController.js

```diff
@@ -42,7 +42,8 @@ exports.getComments = async (req, res) => {
 
 exports.create = async (req, res) => {
     try {
-        const { username, passwordHash, preferredName, email, score } = req.body;
+        const { username, passwordHash, preferredName, email } = req.body;
+        const score = 0;
```

Comment:
Why did you separate score?

Reply:
Passing score is redundant. A new user will have a score of zero. Always.